src/test/java/org/prebid/server/it/YeahmobiTest.java

```diff
@@ -24,8 +24,8 @@
     @Test
     public void openrtb2AuctionShouldRespondWithBidsFromYeahmobi() throws IOException, JSONException {
         // given
-        // AdmixerBidder bid response for imp 001
-        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/yeahmobi-exchange/"))
+        // YeahmobiBidder bid response for imp 001
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/yeahmobi-exchange"))
```

Comment:
Are you sure that /gw-%s-bid.yeahtargeter.com not there ?

Reply:
Not there bc it doesn't override http://localhost:8090/yeahmobi-exchange